source/exercises/mailroom.rst

```diff
@@ -17,6 +17,145 @@ incredibly boring, repetitive emails thanking your donors for their generous
 gifts. You are tired of doing this over and over again, so you've decided to
 let Python help you out of a jam and do your work for you.
 
+Intro Tutorial
```

Comment:
I think it may be better to describe the assignment first, and then provide a tutorial on how to complete at the end.

That way, the students know where they are heading before we start talking about program logic.

Reply:
makes sense, will move it. thanks.